src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -22,14 +22,17 @@ class Focuser(AbstractSerialFocuser):
     as they are marked with the decorator @abstractmethod, we have to override them.
     """
 
-    def __init__(self, name='Astromechanics Focuser', model='Canon EF-232', vendor_id=None,
-                 product_id=None, *args, **kwargs):
+    def __init__(self, name='Astromechanics Focuser', model='Canon EF-232', port=None,
+                 vendor_id=None, product_id=None, *args, **kwargs):
```

Comment:
Can we add the actual `vendor_id` and `product_id` as defaults?

Reply:
My understanding is that it is going through a serial-to-usb converter so this isn't actually querying the astromech (which sort of makes the whole thing pointless) and those converters will all be totally different so it doesn't make sense to add them as defaults.